x-pack/plugins/fleet/server/routes/package_policy/handlers.ts

```diff
@@ -204,6 +204,7 @@ export const createPackagePolicyHandler: FleetRequestHandler<
         pkgName: pkg.name,
         pkgVersion: pkg.version,
         ignoreUnverified: force,
+        skipArchive: true,
```

Comment:
We now need this flag in more places than we don't and its a bit unintuitive, I am going to look to see if we can remove it altogether in a separate issue.